proto/lib/python/slice.proto

```diff
@@ -4,11 +4,11 @@ package syft.lib.python;
 import "proto/core/common/common_object.proto";
 
 message Slice {
-  optional int64 start = 1;
```

Comment:
Q: If we add `optional` (like it was before), doesn't work?

I have seen there was a PR that was referring the `slice`. @tudorcebere do you have any idea if this should be removed or not?

Reply:
The latest protoc (which we are using) uses this, so it's not a problem.

Reply:
Could you put back the optional field in this case?

Reply:
I was testing out optional but it doesnt actually give us what i wanted so i think in the end this might be "optional" lol, aka not needed, because for ints the default is 0 and for bool the default is False, so we only use fields that have their bools set, there are tests so if the tests pass this should be working.